lib/index.ts

```diff
@@ -12,6 +12,28 @@ import {
 
 import { RichtextResolver } from "storyblok-js-client";
 
+import { type SbRichTextOptions as stdSbRichTextOptions} from "@storyblok/richtext";
+
+/**
+ * This is a temporaly class to avoid type collision with the legacy richtext resolver.
+ * It will become ~~`newSbRichTextOptions`~~ -> `SbRichTextOptions` on v4.x
+ */
+export type newSbRichTextOptions = stdSbRichTextOptions;
```

Comment:
Not too strict, but can we have the exports at the end? To have them all located and organized in the same area.